docs/source/usage/plugins/hdf5.rst

```diff
@@ -44,18 +44,26 @@ One can e.g. disable the output of particles by setting:
 ^^^^^^^^^
 
 You can use ``--hdf5.period`` and ``--hdf5.file`` to specify the output period and path and name of the created fileset.
-For example, ``--hdf5.period 128 --hdf5.file simData`` will write the simulation data to files of the form ``simData_0.h5``, ``simData_128.h5`` in the default simulation output directory every 128 steps.
+For example, ``--hdf5.period 128 --hdf5.file simData --hdf5.source 'species_all'`` will write only the particle species data to files of the form ``simData_0.h5``, ``simData_128.h5`` in the default simulation output directory every 128 steps.
 Note that this plugin will only be available if libSplash and HDF5 is found during compile configuration.
 
-============================ ======================================================================================
+============================ ================================================================================================================
 PIConGPU command line option Description
-============================ ======================================================================================
+============================ ================================================================================================================
 ``--hdf5.period``            Period after which simulation data should be stored on disk.
-                             Default is ``0``, which means that no data is stored.
 ``--hdf5.file``              Relative or absolute fileset prefix for simulation data.
                              If relative, files are stored under ``simOutput/``.
-                             Default is ``h5``.
-============================ ======================================================================================
+``--hdf5.source``            Select data sources to dump. Default is ``species_all,fields_all``, which dumps all fields and particle species.
```

Comment:
feel free to continue on the next line (in the 2nd column) after the comma in a table